src/components/FormikFields/FormikInputField/styled.ts

```diff
@@ -0,0 +1,12 @@
+import styled from 'styled-components';
+
+export const EventError = styled.div`
```

Comment:
Kan man bruke Formik sin komponent for ErrorMessage?
https://formik.org/docs/api/errormessage